src/key_value/key_value_client.cc

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) 2021, USC
+// All rights reserved.
+
+#include "key_value_client.h"
+
+#include <glog/logging.h>
+
+namespace csci499 {
+using grpc::ClientContext;
+using grpc::ClientReaderWriter;
+using grpc::Status;
+
+using kvstore::GetReply;
+using kvstore::GetRequest;
+using kvstore::KeyValueStore;
+using kvstore::PutReply;
+using kvstore::PutRequest;
+using kvstore::RemoveReply;
+using kvstore::RemoveRequest;
+
+void KeyValueClient::Put(const std::string& key, const std::string& value) {
+  PutRequest request;
+  request.set_key(key);
+  request.set_value(value);
+
+  PutReply reply;
+  ClientContext context;
+  Status status = stub_->put(&context, request, &reply);
+  if (status.ok()) {
+    LOG(INFO) << "Put request successful from client for key " << key;
```

Comment:
You might want to provide a return value about what happened (at least a bool).

Reply:
fixed in commits